app/models/cart.rb

```diff
@@ -2,21 +2,22 @@ class Cart
   attr_reader :contents
 
   def initialize(initial_contents)
-    @contents = initial_contents || {}
+    @contents = initial_contents || []
   end
 
-  def add_product(product_id, quantity)
-    contents[product_id.to_s] ||= 0
-    contents[product_id.to_s] += quantity.to_i
+  def add_tickets(tickets)
+    tickets.map do |ticket|
+      contents << ticket.id.to_s
```

Comment:
is there a reason why this is a map? also do you need to convert ticket.id to a string?